docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -185,15 +188,18 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 
 === Serializers
 
-Serialization is primarily used in ehcache to store the data in the non-heap tiers. So whenever the user uses a
-multi-tiered cache, serialization and deserialization happens when the data is moved between the tiers. Ehcache provides
-a serializer that can serialize any `Serializable` type. So if you are using standard `Serializable` types as your keys
-and values, no extra serializers need to be configured as the inbuilt serializer would handle those cases. But if you
-use your own custom types in ehcache that are not serializable, the stock serializer won't be able to perform the
-serialization/deserialization on its own. So you need to provide the serialization logic as well. Inorder to do this you
-can register your own custom serializers to the cache/cache manager. So if you register a serializer for some type of
-data, that serializer would be used to perform all the serialization/deserialization for that type.
-You can configure a set of serializers on the cache manager:
+Serialization is primarily used in ehcache to store the data in the non-heap tiers.
+So whenever the user uses a multi-tiered cache, serialization and deserialization happens when the data is moved to, from and between the tiers.
```

Comment:
True, I'd reword that as '… serialization and deserialization *can happen* when …'